src/it/resources/expected/all_datatypes/objcpp/ITAllDatatypes+Private.h

```diff
@@ -13,7 +13,7 @@ namespace djinni_generated {
 struct AllDatatypes
 {
     using CppType = ::AllDatatypes;
-    using ObjcType = ITAllDatatypes*;
+    using ObjcType = ::ITAllDatatypes*;
```

Comment:
I don't know enough about Objective-C++ to answer this myself without testing it:

`ITAllDatatypes` is an Objective-C datatype. You define that `ITAllDatatype` is in the root namespace (`::`), but afaik Objective-C doesn't know the concept of Namespaces. Is this syntax allowed then? Can you attest that this compiles? 🤔

Reply:
the code is Objective-C++ , note the ++ after the C ;-)  
It happens in the namespace , and getting the Objective-C data type from the global namespace is one important aspect of that PR